ext/intl/php_intl.stub.php

```diff
@@ -377,6 +377,9 @@ function datefmt_format_object($datetime, $format = null, ?string $locale = null
 /** @param int $offset */
 function datefmt_parse(IntlDateFormatter $formatter, string $string, &$offset = null): int|float|false {}
 
+/** @param int $offset */
+function datefmt_parse_tocalendar(IntlDateFormatter $formatter, string $string, &$offset = null): int|float|false {}
```

Comment:
lately, the procedural interface is not extended anymore 🤔  See https://externals.io/message/114473#114673